static/js/activity.js

```diff
@@ -320,23 +321,86 @@ exports.insert_user_into_list = function (user_id) {
     compose_fade.update_one_user_row(elt);
 };
 
+function get_users(search_term) {
+    var all_persons = [];
+    if (search_term.indexOf('') !== -1) {
```

Comment:
To avoid bringing the entire realm's list of people into the sidebar.

Currently, when you search for multiple people, like the search term jack, alice - when you type comma, you go from a list of people matching jack to the entire list of displayed users.

With this feature, if it didn't have this idiom, it would be bringing in the whole realm's list of people when you type in that comma, which I thought we should avoid for performance reasons. There are situations already that are a little bit choppy and I thought it wouldn't be a good idea to bring in the entire list.